ci-operator/step-registry/baremetalds/metallb-e2e/test/baremetalds-metallb-e2e-test-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds metallb e2e test command ************"
+
+case "${CLUSTER_TYPE}" in
+packet)
+    # shellcheck source=/dev/null
+    source "${SHARED_DIR}/packet-conf.sh"
```

Comment:
minor: In this case you can simplify the code by sourcing directly `packet-conf`, as the cluster type check is embedded

Reply:
done.